### PR TITLE
Added wooden low bridges

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/ra2/issues/329. Depends upon https://github.com/OpenRA/OpenRA/pull/13882.

Also brings back Arctic Circle. It was trying to import `caoutp` so I added that as well in placeholder quality as that wasn't the intent of this, but I still didn't want it missing in that map.

### CODE REVIEW

**Comment by LinuxDonald** [comment]
Can this not get merged yet?

**Comment by abcdefg30** [comment]
It needs to be rebased first.

**Comment by Mailaender** [comment]
```diff
<<<<<<< low-bridges
ENGINE_VERSION="d5f63f256a74263166bf442394c2e0661c9457e4"
=======
ENGINE_VERSION="2a3a27"
>>>>>>> master
```

I have no idea what is the right version. Those hashes are just cryptic to me.

**Comment by Mailaender** [comment]
Can https://github.com/OpenRA/ra2/pull/393/commits/45594e5cb0dbf99accdabbe697f6ec3e73c41078 be removed now?

**Comment by LinuxDonald** [comment]
Maybe Engine version is git commit hash?

**Comment by Mailaender** [comment]
Rebased though I temporarily had to re-add https://github.com/OpenRA/OpenRA/pull/13802 but exclude the attack move.

**Comment by LinuxDonald** [comment]
Anyone up for review this? :(

**Comment by Mailaender** [comment]
There is a problem with the commandline checks that I don't understand:

```
OpenRA.Utility(1,1): Error: Failed with exception: System.IO.FileNotFoundException: File not found: ^maps/ra2/{DEV_VERSION}
File name: '^maps/ra2/{DEV_VERSION}'
  at OpenRA.FileSystem.FileSystem.Open (System.String filename) [0x00031] in /home/travis/build/OpenRA/ra2/engine/OpenRA.Game/FileSystem/FileSystem.cs:216 
  at OpenRA.FileSystem.FileSystem.OpenPackage (System.String filename) [0x00051] in /home/travis/build/OpenRA/ra2/engine/OpenRA.Game/FileSystem/FileSystem.cs:74 
  at OpenRA.Mods.Common.UtilityCommands.CheckYaml.OpenRA.IUtilityCommand.Run (OpenRA.Utility utility, System.String[] args) [0x001f3] in /home/travis/build/OpenRA/ra2/engine/OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs:93 
make: *** [test] Error 1
```

**Comment by DylanC** [comment]
@abcdefg30 - Can @Mailaender get help with this error? (please)

**Comment by abcdefg30** [comment]
Sorry, I don't know when I find the time to focus on RA2, especially as I'm occupied with "real life" issues at least until the end of February.

**Comment by DylanC** [comment]
@abcdefg30 - No problem. Thanks for the reply.

**Comment by Mailaender** [comment]
@abcdefg30 silently added the tech outpost in https://github.com/OpenRA/ra2/commit/0490d781827b92c94cbb9661ffcfe2bf293e6a8f so I removed my placeholder here. Thanks to @MustaphaTR we have a command bar https://github.com/OpenRA/ra2/pull/451 and I could safely remove the UnitCommandWidget.

I now updated my branch to the next playtest. This does not only fix the bridge SHPs, but also the crate effect animations so I enabled those as well.

**Comment by Mailaender** [comment]
I only moved this a few commits forward now as I faced a wall of actor rules that need manual corrections after the upgrade.

**Comment by Mailaender** [comment]
Also managed to find the fix for https://github.com/OpenRA/OpenRA/issues/14936 on my own.

File: mods/ra2/sequences/misc.yaml [hunk 1]
```diff
@@ -56,19 +56,17 @@ beacon:
 		Length: 1
 		ZOffset: 2047
 
-crate-effects: #Temp Null Some SHPs cannot be read by openra at this time, expect more
```
Comment (MustaphaTR) [inline mods/ra2/sequences/misc.yaml]: You deleted the whole line instead of just the comment.

**Comment by MustaphaTR** [comment]
I'm not sure if it is the same thing @abcdefg30 is compaining on update engine to lastest release PR, but here too, sounds are broken. Everthing that is in audio.bag plays (i think) the chorno leginnerrie fire sound.

File: mods/ra2/sequences/misc.yaml (hunk 1 above)
Comment (Mailaender) [reply mods/ra2/sequences/misc.yaml]: Fixed.

**Comment by abcdefg30** [comment]
I already took enough of your time with this, so I just went ahead and did the last fixes myself:
- Removed some redundant values from the new map (like `Health: 100`)
- Allow huts to connect from two cells distance (like in the original) which makes all bridges work again

Sorry again for the delay.

**Comment by Mailaender** [comment]
Thanks for the assistance. I was going a bit mad with the bridge hut connectivity.

**Pull request merged**